### ISSUE TITLE
Docker build issue finding libcudart

### ISSUE THREAD

**Issue by bolton-lab** [deepchem/deepchem#2467]
## ❓ Questions & Help

After I build the Docker image using your Dockerfile (I am loading into an interactive LSF session with TeslaV100) I get the following error when importing tensorflow:
```
W tensorflow/stream_executor/platform/default/dso_loader.cc:60] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /usr/local/nvidia/lib:/usr/local/nvidia/lib64
2021-04-11 09:17:26.012963: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
```

**Comment by rbharath** [comment]
Hmm, I think this might be related to https://github.com/deepchem/deepchem/issues/2451? I think there are some issues with CUDA11 that people have been running into.

CC @nissy-dev who might have some ideas on what could be happening

**Comment by bolton-lab** [comment]
Yes, i saw that ticket but seems I did not scroll over all the way.  Looks related.  I checked the /usr/local/cuda/lib64 folder however and the libcudart.so is version 10.1.  Also not sure why my error specifically mentions `/usr/local/nvidia/lib:/usr/local/nvidia/lib64` as these paths don't exist anywhere?  I'll keep looking around.

**Comment by nissy-dev** [comment]
TF 2.4 depends on CUDA 11.0 and cudnn 8, so we need to update the base image from `nvidia/cuda:10.1-cudnn7-devel` to `nvidia/cuda:11.0-cudnn8-devel-ubuntu18.04`.

**Comment by bolton-lab** [comment]
great I will try that tomorrow.   Thanks!
